docs/pl/docs/features.md

```diff
@@ -0,0 +1,200 @@
+# Cechy
+
+## Cechy FastAPI
+
+**FastAPI** zapewnia Ci następujące korzyści:
+
+### Oparcie o standardy open
+
+* <a href="https://github.com/OAI/OpenAPI-Specification" class="external-link" target="_blank"><strong>OpenAPI</strong></a> do tworzenia API, w tym deklaracji <abbr title="znane również jako: paths, endpoints, routes">ścieżek</abbr> <abbr title="znane również jako metody HTTP, takie jak POST, GET, PUT, DELETE">operacji</abbr>, parametrów, <abbr title="po angielsku: body requests">ciał zapytań</abbr>, bezpieczeństwa, itp.
+* Automatyczna dokumentacja modelu danych za pomocą <a href="https://json-schema.org/" class="external-link" target="_blank"><strong>JSON Schema</strong></a> (ponieważ OpenAPI bazuje na JSON Schema).
+* Zaprojektowane wokół tych standardów, po drobiazgowej analizie, zamiast skupiania się na górnej warstwie.
```

Comment:
To be honest, I was struggling with this one 😄
Your suggestion is much better, thanks!

Also, the thing about an exact translation - IMO It's fine to not provide an exact translation if it sounds unnatural, and the meaning of the original message is the same (or even better). But the danger is that it can be subjective - which is why I'm glad we have code review process 😁